torch/onnx/_internal/diagnostics/infra/context.py

```
@@ -193,7 +197,7 @@ def __enter__(self):
         return self
 
     def __exit__(self, exc_type, exc_val, exc_tb):
-        return True
+        return None
```

What does this change? Just for my learning

---

Returning `True` will suppress the exception. `None` will allow it to propagate.

---

Thanks!